README.md

```diff
@@ -57,7 +57,7 @@ Prometheus will now be reachable at http://localhost:9090/.
 ### Building from source
 
 To build Prometheus from the source code yourself you need to have a working
-Go environment with [version 1.9 or greater installed](http://golang.org/doc/install).
+Go environment with [version 1.10 or greater installed](http://golang.org/doc/install).
```

Comment:
Do we actually depend on 1.10?

Reply:
1.10 adds  strict json unmarhsaling via `DisallowUnknownFields` which is needed to fix
https://github.com/prometheus/prometheus/pull/3614
otherwise when this PR is merged it won't compile on 1.9 as this function doesn't exist.

Reply:
Makes sense to me.

Reply:
just need someone to spend some time on adding the prometheus golang 1.10 builder :smile:  :thinking: 
https://github.com/prometheus/golang-builder/issues/35

Reply:
I had a moment and was curious so added it https://github.com/prometheus/golang-builder/issues/36